blocks/identity-block/features/account-management/default.jsx

```diff
@@ -82,10 +85,11 @@ function AccountManagement({ customFields }) {
   return (
     <AccountManagementPresentational header={header}>
       {
-        showEmail && (
-          <EmailEditableFieldContainer email={email} onSubmit={handleEmailUpdate} />
-        )
+        showEmail ? (
+          <EmailEditableFieldContainer email={email} />
+        ) : null
       }
+      {showPassword ? <PasswordEditableFieldContainer email={email} hasPassword={hasPassword} /> : null}
```

Comment:
passing in email to PassWordEditableFieldContainer?

Reply:
Yes, as if they don't have a password set you need the email to create their password account